tools/build_defs/scala/scala.bzl

```diff
@@ -26,7 +26,7 @@ load(
 def warning(rule):
   return """This rule has moved out of @bazel_tools!
 
-The scala rules have moved to https://github.com/bazelbuild/scala_rules, you
+The scala rules have moved to https://github.com/bazelbuild/rules_scala, you
 should now refer them via @io_bazel_rules_scala, use:
 
```

Comment:
@johnynek Mhm... I think so, too.

@damienmg Can you have a look at https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/scala/scala.bzl? I think you made those changes and probably know best what they're supposed to look like :)